test/families/location_scale_low_rank.jl

```diff
@@ -0,0 +1,148 @@
+
+@testset "interface LocationScaleLowRank" begin
+    @testset "$(basedist) rank=$(rank) $(realtype)" for
+        basedist = [:gaussian],
```

Comment:
I think something with skew would work, but I think in that case it would only make sense to also learn the skew, which is not quite supported for `LocationScale`.  Maybe `LocationScale` could be extended to support skew. If that is not the case, we can always make a new type that would work for that.

But I do know that there currently is a bug where `mean` and `var` is not correct unless `basedist` is standardized. I'll have to fix this in a different PR and maybe add some non-Gaussian test cases then.